frontend/src/components/calendar/index.jsx

```diff
@@ -7,10 +7,28 @@ import Loading from '../common/loading';
 
 import './styles/index.scss';
 
+function today() {
+    const now = new Date();
```

Comment:
There is a better [DateTime formatting API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) in browsers these days but AFAIK it doesn't give you the strftime level control we need here